docs/spec-conformance.md

```diff
@@ -9,6 +9,7 @@ Spec version | Feature                                  | PR
 -------------|------------------------------------------|----------------------------------------------------------
 v1.1.0       | `SECCOMP_FILTER_FLAG_WAIT_KILLABLE_RECV` | [#3862](https://github.com/opencontainers/runc/pull/3862)
 v1.1.0       | `.process.ioPriority`                    | [#3783](https://github.com/opencontainers/runc/pull/3783)
+v1.1.0       | rsvd hugetlb cgroup                      | TODO ([#3859](https://github.com/opencontainers/runc/issues/3859))
```

Comment:
This should be a separate PR?

Reply:
Seems to have been implemented in:
- https://github.com/opencontainers/runc/pull/4073